.github/workflows/tag.yml

```diff
@@ -50,6 +50,7 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
           token: ${{ secrets.BROADBOT_TOKEN }} # this allows the push to succeed later
+          persist-credentials: false
```

Comment:
I'm not sure about this one given the previous line

Reply:
I think we can always roll it back if necessary.